dbt/include/trino/macros/adapters.sql

```diff
@@ -89,8 +89,12 @@
 {%- endmacro -%}
 
 {% macro comment(comment) %}
-  {%- if comment is not none and comment|length > 0 -%}
-      comment '{{ comment | replace("'", "''") }}'
+  {%- set persist_docs = model['config'].get('persist_docs', {}) -%}
```

Comment:
```suggestion
  {%- set persist_docs = model['config'].get('persist_docs') -%}
```
Here, `get` returns `{}` by default if given key is missing, no need to specify it explicitly